crates/forge/tests/integration/test_state.rs

```diff
@@ -0,0 +1,134 @@
+use std::path::Path;
+
+use crate::integration::common::runner::Contract;
+use crate::integration::common::running_tests::run_test_case;
+use crate::{assert_passed, test_case};
+use indoc::indoc;
+
+#[test]
+fn test_storage_access_from_tests() {
+    let test = test_case!(indoc!(
+        r#"
+        #[starknet::contract]
+        mod Contract {
+            #[storage]
+            struct Storage {
+                balance: felt252, 
+            }
+            
+            #[generate_trait]
+            impl InternalImpl of InternalTrait {
+                fn internal_function(self: @ContractState) -> felt252 {
+                    self.balance.read()
+                }
+            }
+        }
+
+        use ___PREFIX_FOR_PACKAGE___test_case::Contract::balance::InternalContractMemberStateTrait;
```

Comment:
`___PREFIX_FOR_PACKAGE___` 👀

Reply:
It is generated by compiler

Reply:
It is a hack, but it has to be this way in integration tests